### PR TITLE
💄(frontend) improve sale tunnel overriding

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose

Previously, it was pretty hard to customize theme within sale tunnel, and it
could lead to poor result across our sites. Now, we are able to customize
button style within sale tunnel.

## Proposal

- [x] Create a sale tunnel scheme

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -16,26 +16,45 @@ $ make migrate
 
 ## Unreleased
 
+## 2.14.x to 2.15.x
+- RegisteredAddress has now a related stylesheet, you have to update the main stylesheet as follow:
```
Comment (sampaccoud) [inline UPGRADE.md:20]:
```suggestion
- RegisteredAddress now has a related stylesheet, you have to update the main stylesheet as follows:
```

File: src/frontend/scss/tools/_colors.scss [hunk 2]
```diff
@@ -305,6 +305,23 @@
   }
 }
 
+///
+/// An util to apply a scheme to a button
```
[the remark this answers is not in the document]
Comment (jbpenrath) [reply src/frontend/scss/tools/_colors.scss:309]: I took the opportunity to clear my mind about this topic.

File: src/frontend/js/components/SaleTunnel/_styles.scss [hunk 3]
```diff
@@ -8,4 +8,47 @@
...
+    $outline: r-theme-val(sale-tunnel, primary-button-is-outlined);
+    &:not(.disabled) {
+      @include r-button-apply-scheme($scheme, $apply-border: true, $outline: $outline);
+
+      &:hover {
+        @include r-button-apply-scheme($scheme, $apply-border: true, $prefix: 'hover');
+      }
+    }
+    &.disabled {
+      @include r-button-apply-scheme($scheme, $apply-border: true, $outline: $outline);
```
[the remark this answers is not in the document]
Comment (jbpenrath) [reply src/frontend/js/components/SaleTunnel/_styles.scss:23]: I did some checks and yes it appears we are able to apply this scheme as default.

File: src/frontend/js/components/AddressesManagement/_styles.scss [hunk 4]
```diff
@@ -10,7 +10,7 @@
   & > .button {
     .button__icon {
       display: inline-block;
-      fill: none;
+      fill: none !important;
```
[the remark this answers is not in the document]
Comment (jbpenrath) [reply src/frontend/js/components/AddressesManagement/_styles.scss:13]: I found one 👍

[the remark this answers is not in the document]
Comment (sampaccoud) [reply src/frontend/js/components/AddressesManagement/_styles.scss]: :muscle:

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**